openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        period:
+          description: Revenue month.
+          type: string
+          pattern: '^\d{4}-\d{2}$'
```

Even a simple example here would be helpful
```suggestion
          pattern: '^\d{4}-\d{2}$'
          example: 2022-06
```

---

Good point, added, please re-review.